ext/websocket/autobahn/fuzzingclient.js

```diff
@@ -11,7 +11,8 @@ const AUTOBAHN_TESTSUITE_DOCKER =
   "crossbario/autobahn-testsuite:0.8.2@sha256:5d4ba3aa7d6ab2fdbf6606f3f4ecbe4b66f205ce1cbc176d6cdf650157e52242";
 
 const self = Deno.execPath();
-$`${self} run -A --unstable ${pwd}/autobahn_server.js`.spawn();
+$`${self} run -A --unstable --config tests/config/deno.json ${pwd}/autobahn_server.js`
```

Comment:
Maybe compute path for config? I don't think this works right now

Reply:
Passes in CI. Check the `Autobahn testsuite` step [here](https://github.com/denoland/deno/actions/runs/9951572717/job/27491323435?pr=24543).

Reply:
That's what I'm afraid of - it works on CI because it's rooted in the repo root. But if someone was to run that file from different location it would not resolve the config file - I believe that's why `${pwd}/autobahn_server.js` is used here

Reply:
Fixed